trilium-note/umbrel-app.yml

```diff
@@ -0,0 +1,42 @@
+manifestVersion: 1
+id: trilium-note
+category: Productivity
+name: Trilium Note
+version: "0.55.1"
+tagline: Build your personal knowledge base with Trilium Notes
+description: >-
+  Notes can be arranged into arbitrarily deep tree. Single note can be placed into multiple places in the tree (see cloning)
+  Rich WYSIWYG note editing including e.g. tables, images and math with markdown autoformat
+  Support for editing notes with source code, including syntax highlighting
+  Fast and easy navigation between notes, full text search and note hoisting
+  Seamless note versioning
+  Note attributes can be used for note organization, querying and advanced scripting
+  Synchronization with self-hosted sync server
+  there's a 3rd party service for hosting synchronisation server
+  Sharing (publishing) notes to public internet
+  Strong note encryption with per-note granularity
+  Sketching diagrams with bult-in Excalidraw (note type "canvas")
+  Relation maps and link maps for visualizing notes and their relations
+  Scripting - see Advanced showcases
+  REST API for automation
+  Scales well in both usability and performance upwards of 100 000 notes
+  Touch optimized mobile frontend for smartphones and tablets
+  Night theme
+  Evernote and Markdown import & export
+  Web Clipper for easy saving of web content
+developer: ZAdam
+website: https://github.com/zadam/trilium
+dependencies: []
+repo: https://github.com/zadam/trilium
+support: https://github.com/zadam/trilium
+port: 8080
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: /#root
+defaultUsername: ""
+defaultPassword: ""
+torOnly: false
+permissions:
+  - STORAGE_DOWNLOADS
```

Comment:
If shared storage is not required, then this should be dropped.

Reply:
But shared storage helps in multi-device support which makes it unique and is the Notion-like feature which makes it stand out. I would love to keep this here.

Reply:
What do you mean by multi-device support?

The variable `STORAGE_DOWNLOADS` isn't actually implemented/handled by Umbrel (yet), so this doesn't actually do anything atm.